src/main/java/TaskManager.java

```diff
@@ -0,0 +1,82 @@
+public class TaskManager {
+    private Task[] tasks;
+    private int count;
+    private boolean isTaskAdded;
+
+    public TaskManager(int capacity) {
+        tasks = new Task[capacity];
+        this.count = 0;
+        this.isTaskAdded = false;
+    }
+
+    public int getCount() {
+        return count;
+    }
+
+    public boolean getIsTaskAdded() {
```

Comment:
```suggestion
    public boolean IsTaskAdded() {
```

The boolean getter name getIsTaskAdded() doesn’t follow Java naming conventions.